sdk/monitor/Azure.Monitor.Ingestion/tests.yml

```diff
@@ -0,0 +1,10 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      ServiceDirectory: monitor
+      Artifacts:
+        - name: azure-monitor-ingestion
```

Comment:
name and safename should use the .NET naming style used here - https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/monitor/ci.yml#L44